packages/devextreme/js/ui/tabs.js

```diff
@@ -146,15 +148,24 @@ const Tabs = CollectionWidget.inherit({
                 }
             },
             {
-                device: function() {
+                device() {
                     return devices.real().deviceType === 'desktop' && !devices.isSimulator();
                 },
                 options: {
                     focusStateEnabled: true
                 }
             },
             {
-                device: function() {
+                device() {
+                    return isFluent(themeName);
+                },
+                options: {
+                    iconPosition: ICON_POSITION.top,
+                    stylingMode: STYLING_MODE.secondary,
```

Comment:
What about TabPanel component? Why we don't set `secondary` stylingMode for Fluent theme there?

Reply:
[[feat(tabPanel): Add defaultOptions for Material && Fluent](https://github.com/DevExpress/DevExtreme/pull/25734/commits/3a4a7eacdfaae40ea5d66f51381eabb3dd664a96)]